.gitattributes

```diff
@@ -1,6 +1,7 @@
 * text=auto
 
 /.github export-ignore
+/bin export-ignore
```

Comment:
Isn't it necessary to have this in the export, otherwise when installing it as a dependency it'd be impossible to run the `blade-icons-generate` binary? 🤔

Reply:
Oh, this change doesn't seem to exist. Commits seem to just be in a weird order. 😕

- cafad03743f337ba5f59621ae7c7ba23434832d8 - 12 hours ago
- 71039c58ed03f56e9cdbc7ec841dfc45b9b4f7ec - 1 hour ago

But the first seems to override this. Oh well, seems to all be fine in the diff.

Reply:
huh, you're right. What's going on here 🤨

Reply:
Nuked that commit